lib/mixlib/shellout.rb

```diff
@@ -190,19 +195,6 @@ def umask=(new_umask)
       @umask = (new_umask.respond_to?(:oct) ? new_umask.oct : new_umask.to_i) & 007777
     end
 
-    # The uid that the subprocess will switch to. If the user attribute was
-    # given as a username, it is converted to a uid by Etc.getpwnam
-    def uid
```

Comment:
You've moved this to the unix specific stuff, but not done the same for windows which is why the tests are failing.

I don't think Windows has a uid, but the spec says that shellout will always responds to uid